src/PySXO/workflows.py

```diff
@@ -47,18 +52,23 @@ def import_workflow(self, workflow_id, workflow, create_new=False, overwrite=Tru
         }
 
     @cache('_all')
-    def all(self, **kwargs):
-        return [Workflow(self._sxo, i) for i in self._sxo._post(url=f"/v1.1/workflows", **kwargs)]
+    def all(self, **kwargs) -> List[Workflow]:
+        results = list()
+        for page in self._sxo._paginated_request(url=f"/api/v1.1/workflows", **kwargs):
+            results += page
```

Comment:
Use list comprehension, something like that
```suggestion
        results = [page for page in self._sxo._paginated_request(url=f"/api/v1.1/workflows", **kwargs)]

```

Reply:
Good catch bruh